staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal/versionconverter.go

```diff
@@ -56,10 +57,22 @@ func (v *versionConverter) Convert(object typed.TypedValue, version fieldpath.AP
 		return object, err
 	}
 
+	// Convert to internal
+	internalVersion := schema.GroupVersion{
+		Group:   objectToConvert.GetObjectKind().GroupVersionKind().Group,
+		Version: runtime.APIVersionInternal,
```

Comment:
What does this do for custom resource objects, which have no internal version?

Reply:
https://github.com/kubernetes/kubernetes/pull/70087 plumbed a way for each handler to indicate the in-memory version it should deal with. Can we use that instead?